documentation/assemblies/tracing/assembly-distributed-tracing.adoc

```diff
@@ -19,6 +19,13 @@ Support for tracing is built in to the following components:
 * MirrorMaker 2.0
 * Strimzi Kafka Bridge
 
+[NOTE]
```

Comment:
We haven't mentioned Jaeger yet, so maybe we should add a sentence before the note.

_By default, Strimzi uses Jaeger tracing. It is enabled by setting the `spec.tracing.type` property to `jaeger` in the component configuration._